lighthouse-cli/chrome-launcher.ts

```diff
@@ -44,20 +44,17 @@ export class ChromeLauncher {
   chrome?: childProcess.ChildProcess;
   port: number;
 
-  // We can not use default args here due to support node pre 6.
-  constructor(opts?: {
+  constructor(opts: {
     startingUrl?: string,
     chromeFlags?: Array<string>,
     autoSelectChrome?: Boolean,
     port?: number
-  }) {
-    opts = opts || {};
-
+  } = {}) {
     // choose the first one (default)
-    this.autoSelectChrome = defaults(opts.autoSelectChrome, true);
-    this.startingUrl = defaults(opts.startingUrl, 'about:blank');
-    this.chromeFlags = defaults(opts.chromeFlags, []);
-    this.port = defaults(opts.port, 9222);
+    this.autoSelectChrome = defaults<Boolean>(opts.autoSelectChrome, true);
```

Comment:
why is this casting necessary if these types are defined in the method signature?

is it possible to avoid providing the type twice?